src/amo/css/inc/vars.scss

```diff
@@ -23,14 +23,13 @@ $default-font-family: 'Clear Sans', Helvetica, Arial, sans-serif;
 
 // Font-Size Variables
 $default-line-height: 19px;
-$font-size-default: 14px;
+$font-size-default: 16px;
 $font-size-micro: 10px;
```

Comment:
not sure where we use it, but should `micro` increase to `12px`?

Reply:
Good idea. I'll check, rename it to `$font-size-xs`, and update it to `12px`.